### ISSUE TITLE
The focused element should always be visible

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#2336]
Testing #2145

In this case, I have resized the viewlet area in such a way that the `Start (F5)` button is pushed out of the viewport. The button should not be focusable if it is not visible, alternatively the viewlet area should have a different min-width when the debug viewlet is open or the debug viewlet should get a horiz. scrollbar.

Steps: Shift-tab a lot => you will no longer see the active element once you reach the `Start (F5)` button.

![image](https://cloud.githubusercontent.com/assets/5047891/12581973/5e542ef2-c43a-11e5-8ebd-5a918fe37ed8.png)

**Comment by bpasero** [comment]
I wonder if this is a requirement. For example I have tested on Windows, tabbing into a tree where the focused element is not visible and I do not see the element scroll into view. @stevencl maybe you know the answer here.

**Comment by alexandrudima** [comment]
Yeah, it would also be pretty hard to implement this given we support zoom in like this:

![image](https://cloud.githubusercontent.com/assets/5047891/12752215/bc62bca2-c9c1-11e5-8eb9-1691f456f604.png)

So it's ok with to close :)

**Comment by bpasero** [comment]
:+1:

**Comment by stevencl** [comment]
I don't know the answer in terms of accessibility. It does feel odd that a button can be focusable if it is not visible.

For the specific scenario listed above I think we should consider other ways to deal with command placement when resizing the viewlet. But that is for another bug.
